libs/libjson/CMakeLists.txt

```diff
@@ -1,38 +1,5 @@
 # ADDING JSON
 
-set ( JSON_HEADERS
```

Comment:
I see. So, you propose to revert the change on `json_files`, so all headers are also included in it?